README.md

```diff
@@ -1714,6 +1718,8 @@ hz.getReplicatedMap('my-replicated-map').then(function (rmp) {
 });
 ```
 
+Hazelcast Replicated Map uses `EntryListener` to listen to events which occur when entries are added to, updated in, evicted from or removed from the Replicated Map. See the [Entry Listener section](#7522-entry-listener) for information on how to create an entry listener object and register it.
```

Comment:
Hazelcast Replicated Map uses `EntryListener` to listen to the events that occur when the entries are added to, updated in or evicted/removed from the Replicated Map.